src/Orbits.jl

```diff
@@ -55,7 +54,7 @@ the fixed orbital velocity for a circular orbit.
 # Output:
 - v_orb: the average orbital velocity [km/s]
 """
-function average_orbital_velocity(;m_1, m_2, a)
+function circular_orbital_velocity(;m_1, m_2, a)
```

Comment:
Quick comment, this is still not the orbital velocity. It is the relative velocity of both components. Not sure if orbital velocity is a clearly defined concept, but I would interpret it as the velocity of each star in the COM frame, which is different for each.